ftw/simplelayout/browser/resources/main.js

```diff
@@ -6,7 +6,9 @@
     var isUploading = function() {
       return global["xhr_" + $(".main-uploader").attr("id")]._filesInProgress > 0;
     };
-
+    var markBrokenLinks = function(blocks) {
+      $('a.internal-link[href*="resolveuid"]', $(blocks)).addClass("broken-link");
```

Comment:
It seems that ``` blocks ``` are already DOM or jQuery elements otherwise you can name it ``` selector ```. So according to http://api.jquery.com/jquery/ there no need to wrap a DOM or jQuery element again for using it as context for a jQuery selector. So theres no need to wrap ``` blocks ``` again.